server/src/main/java/org/schoellerfamily/pathfinderhelper/datamodel/Character.java

```diff
@@ -123,6 +118,71 @@ public static int getModifier(final Integer abilityScore) {
         }
     }
 
+    /**
+     * @param <T> datatype returned
+     * @param attrname the name of the attribute to be found
+     * @param clazz the class of the attribute
+     * @return the value of the attribute
+     */
+    public <T> T getAttribute(final String attrname, final Class<T> clazz) {
```

Comment:
Do you have a specific use in mind for this method? Otherwise, perhaps you could just do getStringAttribute and getIntegerAttribute. In which case the analysis of class by its name can go away.

There should be some explicit tests of these new ways to get at attributes.